src/Codeception/Module/REST.php

```diff
@@ -333,6 +334,37 @@ public function amNTLMAuthenticated($username, $password)
     }
 
     /**
+     * Allows to send REST request using AWS Authorization
+     * Requires client to by Guzzle
```

Comment:
It's `be`, not `by`.
Also that we are using Guzzle HTTP Client is an internal detail that some users may be unaware of.
I think that it would be better to write `Only works with PhpBrowser`.

Reply:
Updated the Docblock